client/src/sections/learning-paths/LearnersSection/index.tsx

```diff
@@ -7,7 +7,7 @@ import SortDropdown, {
   type SortOption,
 } from '@/src/shared/components/Dropdown/SortDropdown/SortDropdown';
 import ArrowIcon from '@/src/shared/icons/ArrowIcon';
-import Button from '@/src/shared/components/Button';
+import AddLearnerModal from '../../../shared/components/Modal/AddLearnerModal';
```

Comment:
```suggestion
import AddLearnerModal from './AddLearnerModal';
```

Reply:
I think you created a modal both in modal component folder and sections folder

Reply:
Yes I forgot to delete the AddLearnerModal from section folder.

Reply:
why you use the modal from shared components instead of the one that is in sections folder, may I know why?

Reply:
This was the comment of sir Vali so that, to make it reusable as well.

Reply:
![image](https://github.com/framgia/sph-lms/assets/110363852/ded87c76-4b5f-447b-960d-abc66b34577c)